src/common.go

```diff
@@ -1,12 +1,10 @@
 package main
 
-import "log"
-
 //Panics if `err` is not `nil`
 //
 // @param err Error to be checked for equality to `nil`
 func panicIfError(err error) {
 	if err != nil {
-		log.Panic(err)
+		panic(err)
```

Comment:
Moved logging to middleware, because we can start panicking without calling `panicIfError`